.tekton/git-clone-task.yaml

```diff
@@ -0,0 +1,436 @@
+---
+apiVersion: tekton.dev/v1beta1
```

Comment:
Question: Is there a reason not to go straight to `tekton.dev/v1` instead of `tekton.dev/v1beta1`?
I thought it saves [one more migration](https://tekton.dev/docs/pipelines/migrating-v1beta1-to-v1/#migrating-from-tekton-v1beta1-to-tekton-v1) if  `tekton.dev/v1` is used from the beginning.

Reply:
Good point. I was looking at the examples in the https://github.com/open-toolchain/tekton-catalog repository. They use v1beta. I will switch to v1 👍 Thank you!

Reply:
Updated.